CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.3.1 (2020-03-11)
+
+- Fixed the country field component to not take up extra space below the component. [#59](https://github.com/blackbaud/skyux-lookup/pull/59)
```

Comment:
"...to eliminate extra space below the component."

Reply:
Done